### PR TITLE
[target] SIRMIXALOT add gyro, baro, flash

### PR DESCRIPTION
**Pull request by nerdCopter**
- according to schematic
- https://oshpark.com/shared_projects/uQkUjylH

### CODE REVIEW

File: configs/SIRMIXALOT/config.h [hunk 1]
```diff
@@ -26,6 +26,18 @@
...
+
+#define USE_ACC
+#define USE_ACC_SPI_ICM20602
+#define USE_GYRO
+#define USE_GYRO_SPI_ICM20602
+#define USE_FLASH
+#define USE_FLASH_W25N01G
+#define USE_BARO
+#define USE_BARO_BMP280
+#define USE_BARO_SPI_BMP280
```
Comment (haslinghuis) [inline configs/SIRMIXALOT/config.h:38]: There is no SPI define for baro, so we need I2C define
```suggestion
```

Comment (nerdCopter) [reply configs/SIRMIXALOT/config.h:38]: sorry, pasted both and did not follow up.

Comment (haslinghuis) [reply configs/SIRMIXALOT/config.h]: Updated the comment. You might have missed that :)

**Review by haslinghuis — APPROVED** [review]

**Comment by ot0tot** [comment]
Is this a community target? Is there a Discord channel for it?

**Comment by nerdCopter** [comment]
@ot0tot , it's a legacy product, not alive.  just want the configs to compile and be accurate according to search-results.

File: configs/SIRMIXALOT/config.h [hunk 2]
```diff
@@ -26,6 +26,16 @@
...
 #define MANUFACTURER_ID   CUST
 
+#define USE_ACC
+#define USE_ACC_SPI_ICM20602
+#define USE_GYRO
+#define USE_GYRO_SPI_ICM20602
+#define USE_FLASH
+#define USE_FLASH_W25N01G
+#define USE_BARO
+#define USE_BARO_BMP280
```
Comment (ot0tot) [inline configs/SIRMIXALOT/config.h:36]: The baro is on SPI3, not I2C.
```suggestion
#define USE_BARO_SPI_BMP280
```
image

Comment (ot0tot) [reply configs/SIRMIXALOT/config.h:36]: Also need to add:
```
#define BARO_CS_PIN          PB9
```

File: configs/SIRMIXALOT/config.h [hunk 3]
```diff
@@ -79,6 +89,7 @@
 
 //TODO #define SERIALRX_INVERTED ON
 //TODO #define SERIALRX_HALFDUPLEX ON
+#define BARO_I2C_INSTANCE (I2CDEV_1)
```
Comment (ot0tot) [inline configs/SIRMIXALOT/config.h:92]:
```suggestion
#define BARO_SPI_INSTANCE SPI3
```

**Review by ot0tot — APPROVED** [review]
I pushed a commit to add `#define BARO_CS_PIN          PB9`. 
Should be good to go now.

**Pull request merged**